dashboard/app/views/layouts/_survey.html.haml

```diff
@@ -0,0 +1,67 @@
+-# If user is seeing English
+- if I18n.locale.to_s.downcase.split('-').first == "en"
```

Comment:
We have a helper `language` that does this that you can use here: https://github.com/code-dot-org/code-dot-org/blob/5214eac47ab2eac67da55db29c10e9883d3381f5/dashboard/app/helpers/locale_helper.rb#L18